emissionscheck_alb/data.py

```diff
@@ -0,0 +1,46 @@
+def keep_pass_fail(df):
+    '''
+    Removes rows that do not have value `"P"` or `"F"` in target variable `OVERALL_RESULT`
+    
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        Data-frame containing the column `OVERALL_RESULT`
+    Returns
+    -------
+    pandas.DataFrame
+        DataFrame of rows with desired values `'P'` or `'F'` in target variable 
+    '''
+    # Mask to select only rows with desired target values
+    overall_result_mask = (df['OVERALL_RESULT'] == 'P') | (df['OVERALL_RESULT'] == 'F')

```

Comment:
this can be shortened to:
```suggestion
    df = df[df.OVERALL_RESULT.isin(['P', 'F'])]

```

Reply:
Done